databases/week3/week3hw.sql

```diff
@@ -0,0 +1,166 @@
+USE meal_sharing;
+
+-- ********************** Meal **********************
+-- Get all meals
+SELECT*
```

Comment:
does it run without a space?
```suggestion
SELECT *
```